lib/rbs/trace/inline_comment_visitor.rb

```diff
@@ -58,6 +58,17 @@ def find_method_definition(name)
           member.is_a?(AST::Members::MethodDefinition) && member.name == name
         end
       end
+
+      def with_context(node)
+        names = node.constant_path.full_name_parts
```

Comment:
When I add the type declaration and try steep check, I get the following error

```
    lib/rbs/trace/inline_comment_visitor.rb:64:35: [error] Type `(::Prism::ConstantReadNode | ::Prism::ConstantPathNode | ::Prism::MissingNode | ::Prism::CallNode)` does not have method `full_name_parts`
    │ Diagnostic ID: Ruby::NoMethod
    │
    └         names = node.constant_path.full_name_parts
                                         ~~~~~~~~~~~~~~~
```

It seems that constant_path may return MissingNode or CallNode.
However, I don't think InlineCommentVisitor needs to support these classes, so there is no problem to make an exception.


```ruby
constant_path = node.constant_path
raise if constant_path.is_a?(Prism::MissingNode) || constant_path.is_a?(Prism::CallNode)

Reply:
Agree with that. I applied your suggestion here: b4d3d2d7479769dc96d261e62243718ab19cd78c